plugins/woocommerce-admin/client/marketing/components/PluginCardBody/SmartPluginCardBody.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { Pill } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { PluginCardBody } from '~/marketing/components';
+import { RecommendedPlugin } from '~/marketing/types';
+import { getInAppPurchaseUrl } from '~/lib/in-app-purchase';
+import { createNoticesFromResponse } from '~/lib/notices';
+import { useIsPluginInstalledNotActivated } from './useIsPluginInstalledNotActivated';
+import './PluginCardBody.scss';
+
+type SmartPluginCardBodyProps = {
```

Comment:
I would personally rethinking about the naming for this. Smart seems a bit generic